spark-jobs/src/test/scala/za/co/absa/enceladus/conformance/interpreter/fixtures/StreamingFixture.scala

```diff
@@ -51,6 +50,8 @@ trait StreamingFixture extends FunSuite with SparkTestBase with MockitoSugar {
     when(configStub.containsKey(menasUriKey)).thenReturn(true)
     when(configStub.getString(menasUriKey)).thenReturn("https://mymenas.org")
     when(configStub.containsKey(menasAuthKeytabKey)).thenReturn(true)
+    when(configStub.containsKey(menasCredentialsFileKey)).thenReturn(false)
```

Comment:
explicit stub definition instead of relying on the false-returning default behavior.